boards: nrf: Indicate watchdog as supported on Nordic DK boards

anangl opened the pull request:
Indicate that watchdog is supported on several Nordic DK boards
so that the wdt_nrfx driver is covered by CI builds.

Related to #12860.

Signed-off-by: Andrzej Głąbek <andrzej.glabek@nordicsemi.no>

anangl committed:
boards: nrf: Indicate watchdog as supported on Nordic DK boards

Indicate that watchdog is supported on several Nordic DK boards
so that the wdt_nrfx driver is covered by CI builds.

Signed-off-by: Andrzej Głąbek <andrzej.glabek@nordicsemi.no>

ioannisg commented on boards/arm/nrf9160_pca10090/nrf9160_pca10090.yaml:
```diff
@@ -9,3 +9,4 @@ ram: 64
 flash: 256
 supported:
   - i2c
+  - watchdog
```
Will there be a problem that wdt is not enabled in DTS?

anangl replied:
Hmm, I haven't thought about this. I've just run the builds and they went fine.
The "status" property for the `wdt` node [is not specified](https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/nordic/nrf9160_common.dtsi#L217) in _nrf9160_common.dtsi_ and the node seems to be enabled by default.

ioannisg replied:
So: does this mean that if the "status" property is missing, the default is "ok"?

ioannisg replied:
My point is that we need the device to be enabled, to get the NRF_WDT defined (at the moment) for nRF9160_pca10090/ns boards. I suppose, any WDT-related sample requires the register base address be defined...

ioannisg replied:
IF the "default" is "status ok", then I would prefer to actually add the property, so it is more visible to the reader.

anangl committed:
dts: nordic: Enable wdt nodes by default for all nRF SoCs

Actually, add the "status" property that enables the nodes explicitly.
They were apparently enabled by default without this property.

Signed-off-by: Andrzej Głąbek <andrzej.glabek@nordicsemi.no>
